tpl/page_optm/settings_html.tpl.php

```diff
@@ -87,6 +87,26 @@
 			</td>
 		</tr>
 
+		<tr>
+			<th>
+				<?php $id = Base::O_OPTM_HTML_SKIP_COMMENTS; ?>
+				<?php $this->title($id); ?>
+			</th>
+			<td>
+				<?php $this->build_textarea($id); ?>
+				<div class="litespeed-desc">
+					<?php echo __('Keep html comments.', 'litespeed-cache'); ?>
+					<br />
+					<?php echo __('If comment to be kept is like:', 'litespeed-cache'); ?>
+					<code>&lt;!-- A comment that need to be here --&gt;</code>
```

Comment:
`A comment that needs to be here`

Reply:
Changed the text, please confirm is ok :) thank you